aries_cloudagent/config/argparse.py

```diff
@@ -1439,4 +1465,18 @@ def get_settings(self, args: Namespace):
 
             if args.multitenant_admin:
                 settings["multitenant.admin_enabled"] = True
+
+            if args.multitenancy_config:
+                multitenancyConfig = json.loads(args.multitenancy_config)
```

Comment:
There may be a more consistent way to support these options between the YAML argument handling in other places and the generic plugin configuration mechanism, but I'm okay merging this as-is for now. @dbluhm